### DISCUSSION TITLE
Support extension re-naming for sidecar caption files

### DISCUSSION THREAD

**Discussion by freddieFishCake** [JarvyJ/pislide-os#15] [Feature Suggestions]
PiSlide-OS currently uses sidecar files with a concatenated '.txt', e.g. an image DSC_01234.jpg could have the sidecar DSC_01234.jpj.txt.
Could we also support replacing the file extension, e.g. DSC_01234.jpg with a caption in sidecar DSC_01234.txt?

**Comment by JarvyJ** [comment]
I have thought about this, there are a few times where folks could get collisions. You have an `example.jpg` and an `example.png` and they both would get the caption from `example.txt`, but in reality I think this is a fairly uncommon edge case.

I guess if enough folks are interested in it, we could make it happen!
